docs/apps/qrscp.rst

```diff
@@ -27,6 +27,10 @@ SOP Instances sent to the application using the Storage service have some of
 their attributes added to a sqlite database that is used to manage Instances
 for the Query/Retrieve service.
 
+In addition, the ``qrscp`` application implements a Service Class Provider (SCP) for the
+:dcm:`Basic Modality Worklist<part04/chapter_K.html>`, and :dcm:`Unified Procedure Step<part04/Chapter_CC>`
+service classes, but currently will only return empty results (0 records)
+
 .. warning::
```

Comment:
Empty results are a legitimate and frequent outcome when issuing a C-FIND-RQ
This happens alot in IHE-RO TDW-II profile because the SCU implementations often poll every five minutes and there isn't a match at the time.
It also happens for typical Q/R and for BWM/MWL. 

However, I am intending to provide a means for the tester supplying UPS to provide  responses to matching queries.
I was going to do that separately from qrscp, and do it in a personal but public repo with TDW-II plus examples. 

The reason I put in the successful empty response was so there could be a basic communication test.  There wasn't one for BWM/WLM.  Which was already supported by findscu. If you want me to remove the minimal support for UPS CFIND from qrscp and will still approve the changes for findscu, I will create a new PR and cancel this one.  But if I have to implement a fully functional example, then we should remove support for BWM/MWL from findscu until it too has a fully functional test available.

I will be happy to transfer any and all of my personal repo work on UPS to pynetdicom apps once I am happy with it.  I realise that counts as big bang PRs, but it can be done one paired set of apps at a time (mutual testing... It takes two to tango).   I have that minimal pair already for UPS Watch, although it is not "feature complete".
I'm also happy to maintain ownership of my TDW-II/UPS working examples.

Reply:
> If you want me to remove the minimal support for UPS CFIND from qrscp and will still approve the changes for findscu, I will create a new PR and cancel this one.

Your first statement that this is legitimate and happens commonly is enough for me, especially with the additional abilities available soon.  As to whether in your own repo, or contributed here, I'll let @mrbean-bremen comment on that one, I don't know enough to have an opinion.

Reply:
Awesome.
There are a handful of RT 3rd party vendors eager to see the example of a findscu that supports  UPS.  Having this in the main (master) branch will make it easier for them.
Thank you